buildenv

```diff
@@ -44,6 +44,20 @@ zopen_install() {
   rm -rf ${ZOPEN_INSTALL_DIR}
   mkdir -p ${ZOPEN_INSTALL_DIR}
   cp -pR out/* ${ZOPEN_INSTALL_DIR}
```

Comment:
I would suggest replacing this PR with the following changes. You essentially want the gn executable in the bin path. And zopen build will then also create the symlinks automatically for you.
```suggestion
  mkdir -p ${ZOPEN_INSTALL_DIR}/bin
  cp -pR out/* ${ZOPEN_INSTALL_DIR}/bin
```
zopen build automatically creates symlinks. I would avoid creating them manually as below.